### ISSUE TITLE
Proposal - switch to node-sass

### ISSUE THREAD

**Issue by cheapsteak** [Jam3/generator-jam3#131]
They are syntax-compatible for 95% of use cases

**Less's `calc` is defective** 

``` sass
width: calc(100% - 100px)
```

This gets compiled to: 
with LESS: [[demo]](http://lesscss.org/less-preview/#%7B%22less%22%3A%22div%20%7B%5Cn%20%20width%3A%20calc%28100%25%20-%20100px%29%5Cn%7D%22%7D)

``` css
width: calc(0%); /*wat*/
```

with SCSS: [[demo]](http://sassmeister.com/gist/b9e164b791730b5590b5)

``` css
width: calc(100% - 100px);
```

You have to do escape the string in less

``` less
width: ~"calc(100% - 100px)";
```

This is mental and has tripped at least one developer on each project I've had  

Mixin definitions are different, but not by much.
(Haven't really had cause to use mixins since autoprefixer)

Loops are different.
Haven't had to use a loop for a long time either, but if you need to, sass's loop syntax is, imo, better.
Sass's has `for` loops and `each` loop that can loop through indexes or arrays of strings
Less uses recursion for loops and can only loop through numbers.

Less was the first viable css preprocessor in node, back when using sass meant either you had to install ruby, or settle for the (at the time) much less powerful libsass. But libsass now has 98.53% feature parity with ruby sass (node-sass is a wrapper for libsass). 

So to sum up, I propose to switch to node-sass (or at least have trial runs where projects are allowed to use node-sass instead of less) because its `calc` isn't defective, the language features we rarely use are different but not by much, and sass is the [industry standard](https://www.google.ca/trends/explore#q=%2Fm%2F054k6n_%2C%20%2Fm%2F0gjd0jv&cmpt=q&tz=Etc%2FGMT%2B4).
